Projects/My_website/myWebsite/blog/apps.py

```diff
@@ -4,3 +4,6 @@
 class BlogConfig(AppConfig):
     default_auto_field = "django.db.models.BigAutoField"
     name = "blog"
+
+    def ready(self):
+        from blog import signals
```

Comment:
@codal-uchoudhary 
Never add import statements in the middle of a file.

All Import statements must be at the top of the file.

Reply:
<img width="450" alt="image" src="https://github.com/codal-uchoudhary/python-training-2024/assets/156191290/34319e28-55fe-4b61-9d8a-659bcee3e87d">

Reply:
here i am using @receiver() decorator , so i have to import the signal file in the redy() function.